src/documents/models.py

```diff
@@ -404,6 +404,9 @@ class SavedViewFilterRule(models.Model):
         (20, _("fulltext query")),
         (21, _("more like this")),
         (22, _("has tags in")),
+        (23, _("ASN greater than")),
```

Comment:
I'm never sure, but this maybe need a migration?

Reply:
Oh youre right because thats an integer 'choice' field. Thanks for pointing that out! 

So this is weird, when I run `makemigrations` it's giving a lot of changes to the comment table, see below. Something specific to my install or is the comment migration messed up somehow?

```
class Migration(migrations.Migration):
    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('documents', '1024_merge_20220824_1341'),
    ]
    operations = [
        migrations.AlterModelOptions(
            name='comment',
            options={'ordering': ('created',), 'verbose_name': 'comment', 'verbose_name_plural': 'comments'},
        ),
        migrations.RemoveField(
            model_name='comment',
            name='document_id',
        ),
        migrations.RemoveField(
            model_name='comment',
            name='user_id',
        ),
        migrations.AddField(
            model_name='comment',
            name='document',
            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='documents', to='documents.document', verbose_name='document'),
        ),
        migrations.AddField(
            model_name='comment',
            name='user',
            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='users', to=settings.AUTH_USER_MODEL, verbose_name='user'),
        ),
        migrations.AlterField(
            model_name='comment',
            name='comment',
            field=models.TextField(blank=True, help_text='Comment for the document', verbose_name='content'),
        ),
        migrations.AlterField(
            model_name='comment',
            name='created',
            field=models.DateTimeField(db_index=True, default=django.utils.timezone.now, verbose_name='created'),
        ),
        migrations.AlterField(
            model_name='savedviewfilterrule',
            name='rule_type',
            field=models.PositiveIntegerField(choices=[(0, 'title contains'), (1, 'content contains'), (2, 'ASN is'), (3, 'correspondent is'), (4, 'document type is'), (5, 'is in inbox'), (6, 'has tag'), (7, 'has any tag'), (8, 'created before'), (9, 'created after'), (10, 'created year is'), (11, 'created month is'), (12, 'created day is'), (13, 'added before'), (14, 'added after'), (15, 'modified before'), (16, 'modified after'), (17, 'does not have tag'), (18, 'does not have ASN'), (19, 'title or content contains'), (20, 'fulltext query'), (21, 'more like this'), (22, 'has tags in'), (23, 'ASN greater than'), (24, 'ASN less than'), (25, 'storage path is')], verbose_name='rule type'),
        ),
    ]
```

Reply:
Oh I see, `document_id` --> `document` foreign key. Should those changes be included in the `1023_add_comments.py` migration though?

Reply:
Hey @stumpylog take a look at my last commit, I tried cleaning up the migrations. I think this is OK since we havent released this yet right? And this seems like how they 'should' be, yea?

Reply:
I'm guessing the model got updated, but the migration wasn't re-created?

I just deleted `1023_add_comments.py` and `1024_merge_20220824_1341.py`, and it made me a new `1024_comment.py` which creates the model with no alter fields.

Reply:
Yeah, that seems right and ok.  We wouldn't if this was released, but that's what I was thinking to do.